content/chainguard/libraries/python/overview.md

```diff
@@ -76,31 +98,100 @@ In some cases, multiple CVEs may be remediated in a specific library version.
+pyyaml-6.0.3-cp312-cp312-manylinux_2_28_aarch64.whl
+pyyaml-6.0.3-cp312-cp312-manylinux_2_28_x86_64.whl
+pyyaml-6.0.3-cp312-cp312-manylinux_2_39_aarch64.whl
+pyyaml-6.0.3-cp312-cp312-manylinux_2_39_x86_64.whl
+pyyaml-6.0.3-cp313-cp313-manylinux_2_28_aarch64.whl
+pyyaml-6.0.3-cp313-cp313-manylinux_2_28_x86_64.whl
+pyyaml-6.0.3-cp313-cp313-manylinux_2_39_aarch64.whl
+pyyaml-6.0.3-cp313-cp313-manylinux_2_39_x86_64.whl
+pyyaml-6.0.3-cp39-cp39-manylinux_2_28_aarch64.whl
+pyyaml-6.0.3-cp39-cp39-manylinux_2_28_x86_64.whl
+pyyaml-6.0.3-cp39-cp39-manylinux_2_39_aarch64.whl
+pyyaml-6.0.3-cp39-cp39-manylinux_2_39_x86_64.whl
+pyyaml-6.0.3.tar.gz
 ```
 
-The first index provides all Python libraries that Chainguard builds from
-source, without remediated versions. The second index provides remediated
-libraries with high and critical CVE fixes applied to older versions. The
-separate indexes provide the option to make remediated versions available for
-your development or opt out of using these versions completely and continue to
-use upstream versions only.
+The different file names use the following qualifiers to allow downloading the
+correct wheel file by the build and packaging tool:
+
+* The values `cp*` indicate CPython compatibility.
+* The first value `cp39`, `cp310`, and others indicates the compatibility with
+  Python version `3.9`, `3.10`, and others.
+* The second value `cp39`, `cp310`, and others indicates the compatibility with
+  the Python Application Binary Interface (ABI) version `3.9`, `3.10`, and
+  others.
+* The `manylinux` value indicates that the wheel is suitable for any modern
+  Linux distribution, since not distribution specific requirements are in place.
+* The `2_28` and `2_39` values indicate suitability for environments using
+  `glibc 2.28` and higher or `glibc 2.39` and higher.
+* The `aarch64` and `x86_64` values indicated the processor architecture
+  requirement for ARM or x86.
+
+As a result of the provided files for these libraries, the following
+requirements apply:
+
+* All Linux distributions must use glibc 2.28 or higher, such as RHEL 8, Ubuntu
+  20.04, or Amazon Linux 2023.
+* On Windows and MacOS you must use a suitable Linux distribution with a
+  container solution, such as WSL2, apple/container or Docker Desktop.
+* Processor architecture of the runtime environment must be `x86_64` or
+  `aarch64`.
+
+### Behavior on Windows and MacOS
+
+As a result of the requirements detailed in the preceding section, the developer
+experience on Windows and MacOS platforms is impacted. Packages that include
+native binaries are provided for Linux platforms only. These are not suitable
+for use on Windows or MacOS.
+
+When using Chainguard Libraries for Python on these platforms,
+retrieval for these libraries and therefore application builds fail. The
+suggested deployment, detailed in [Technical Details](#technical-details) and
+[Global Configuration](/chainguard/libraries/python/global-configuration/), uses
```

Comment:
```suggestion
[Global Configuration](/chainguard/libraries/python/global-configuration/), involves
```

not sure that's the right word, but should avoid using "uses" twice

Reply:
Thats good.